src/libcddb-1-windows-macro-fix.patch

```diff
@@ -0,0 +1,18 @@
+This file is part of MXE. See LICENSE.md for licensing information.
+
+Newer versions winioctl.h contain a #define SEARCH_ALL .. which collides with the enum in cddb_conn.h.
```

Comment:
Please format the patch with patch-tool-mxe.
See https://github.com/mxe/mxe/pull/1277#issuecomment-257083463 for more info on it.